### PR TITLE
Prepare for deploy to kube

### PR DESCRIPTION
**Pull request by jharlap**
CLOUD-635

### CODE REVIEW

File: k8s.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  strategy:
+    rollingUpdate:
+      maxUnavailable: 0
+  template:
+    metadata:
+      labels:
+        app: kamu
+        env: prod
+    spec:
+      revisionHistoryLimit: 1
```
Comment (jessicawoo) [inline k8s.yml:16]: Yay

**Review by jessicawoo — APPROVED** [review]
R+ :shipit:

**Comment by jharlap** [comment]
@minosabaki Updated the build process a bit so isn't as fragile (and actually works now!) - will squash once you've had a chance to re-review.

**Review by jessicawoo — APPROVED** [review]
R+ :shipit:

**Pull request merged**